### PR TITLE
spelling fixes, just comments and readme.

### PR DESCRIPTION
**Pull request by ka7**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -2211,7 +2211,7 @@
   *) An attacker can force an error condition which causes openssl to crash
      whilst processing DTLS packets due to memory being freed twice. This
      can be exploited through a Denial of Service attack.
-     Thanks to Adam Langley and Wan-Teh Chang for discovering and researching
+     Thanks to Adam Langley and Wan-The Chang for discovering and researching
```
Comment (richsalz) [inline CHANGES]: Wrong.

**Comment by richsalz** [comment]
except for one error, looks good.

**Comment by richsalz** [comment]
+1

**Comment by mattcaswell** [comment]
+1

**Comment by kroeckx** [comment]
I think I disabled coveralls from commenting on it.

**Comment by richsalz** [comment]
merged, thanks!

**Pull request closed without merging**